src/t8_forest/t8_forest_cxx.cxx

```diff
@@ -1492,7 +1747,88 @@ t8_forest_copy_trees (t8_forest_t forest, t8_forest_t from, int copy_elements)
   }
 }
 
-/* Search for a linear element id (at forest->maxlevel) in a sorted array of
+/* TODO: To work for many processes, global_num_trees, tree_offsets,
+ * first_local_tree and last_local_tree must be calculated for each process. */
+/* TODO: Extend to nonempty trees. */
```

Comment:
This looks like a rather unfinished function. Either implement it fully, oder move it to another branch or delete it.